src/metax_api/services/catalog_record_service.py

```diff
@@ -52,61 +52,90 @@ def get_queryset_search_params(cls, request):
         Get and validate parameters from request.query_params that will be used for filtering
         in view.get_queryset()
         """
+        queryset_search_params = {}
 
         if not request.query_params:
-            return {}
+            queryset = cls.filter_by_state(request, {})
+            return queryset
 
-        queryset_search_params = {}
+        else:
```

Comment:
The ``else`` block is unnecessary, since the previous ``if`` block always returns. It is better the keep code indentation lower.